src/utils/validateAvaxBurnedAmountEtna.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  isAddPermissionlessDelegatorTx,
+  isAddPermissionlessValidatorTx,
+  isAddSubnetValidatorTx,
+  isCreateChainTx,
+  isCreateSubnetTx,
+  isPvmBaseTx,
+  isExportTx as isPvmExportTx,
+  isImportTx as isPvmImportTx,
+  isRemoveSubnetValidatorTx,
+  isTransferSubnetOwnershipTx,
+} from '../serializable/pvm';
+import type { UnsignedTx } from '../vms/common';
+
+export const validateAvaxBurnedAmountEtna = ({
+  unsignedTx,
+  burnedAmount,
+  baseFee,
+  feeTolerance,
+}: {
+  unsignedTx: UnsignedTx;
+  burnedAmount: bigint;
+  baseFee: bigint; // pvm dynamic fee caculator: @see https://github.com/ava-labs/avalanchego/blob/master/vms/platformvm/txs/fee/dynamic_calculator.go
+  feeTolerance: number; // tolerance percentage range where the burned amount is considered valid. e.g.: with FeeTolerance = 20% -> (expectedFee <= burnedAmount <= expectedFee * 1.2)
```

Comment:
Can you move these comments to JSDoc style comments so they get more exposure without having to dive into this file specifically to come across them.

Reply:
[updated](https://github.com/ava-labs/avalanchejs/pull/880/commits/7502549ed4fcee63a282678d17b321f8be06e201)